polygeist/tools/cgeist/Lib/CGExpr.cc

```diff
@@ -939,196 +939,6 @@ const clang::FunctionDecl *MLIRScanner::EmitCallee(const Expr *E) {
   return nullptr;
 }
 
-std::pair<ValueCategory, bool>
-MLIRScanner::EmitBuiltinOps(clang::CallExpr *expr) {
```

Comment:
Moved to CGCall.cc as it is called from `VisitCallExpr` which is already in that file.